### PR TITLE
Stubbing out a file for Belfast

### PR DESCRIPTION
**Pull request by andymcf**

### CODE REVIEW

**Comment by andymcf** [comment]
This PR resolves #106

File: andymcf-belfast.md [hunk 1]
```diff
@@ -0,0 +1 @@
+##Belfast
```
Comment (IliaIdakiev) [inline andymcf-belfast.md:1]: Is this the capital of Northern Ireland? Looks like a great city 😄

Comment (andymcf) [inline andymcf-belfast.md:1]: 👍 it is

Comment (Gerva1s) [inline andymcf-belfast.md:1]: Excellent spot - when the sun shines

**Pull request merged**